modules/functional-test/src/main/java/com/intuit/wasabi/tests/library/TestBase.java

```diff
@@ -89,4297 +91,4493 @@
 @Listeners({RetryListener.class})
 public class TestBase extends ServiceTestBase {
 
-    private static final Logger LOGGER = getLogger(TestBase.class);
```

Comment:
@sudheerkethireddy it looks like you formatted the code so it created a big DIFF. Can you not format the code if there is no change in the code?